assets/js/vue/mixins/themeFunctions.js

```diff
@@ -1,7 +1,9 @@
-export const themeFunctions = {
+const themeFunctions = {
   methods: {
-    getThemeProperty: function (prop) {
-      return getComputedStyle(document.body).getPropertyValue("--" + prop);
-    }
-  }
-}
+    getThemeProperty(prop) {
+      return getComputedStyle(document.body).getPropertyValue(`--${prop}`);
+    },
+  },
+};
+
+export default themeFunctions;
```

Comment:
This causes a build error. I don't think you can export default and the function. My suggestion makes it work.
```suggestion
export {themeFunctions};
```

Reply:
Fixed now with additional configuration.